kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/core/v1/PodOperationsImpl.java

```diff
@@ -406,38 +407,33 @@ public InputStream read() {
   }
 
   private InputStream readFile(String source) {
-    //Let's wrap the code to a callable inner class to avoid NoClassDef when loading this class.
-    try {
-      return new Callable<InputStream>() {
-        @Override
-        public InputStream call() {
-          try {
-            PipedOutputStream out = new PipedOutputStream();
-            PipedInputStream in = new PipedInputStream(out, 1024);
-            ExecWatch watch = writingOutput(out).usingListener((int code, String reason) -> {
-                try {
-                  out.flush();
-                  out.close();
-                } catch (IOException e) {
-                  e.printStackTrace();
-                }
-              }
-            ).exec("sh", "-c", String.format("cat %s | base64", PodUpload.shellQuote(source)));
-            return new org.apache.commons.codec.binary.Base64InputStream(in) {
-              @Override
-              public void close() throws IOException {
-                watch.close();
-                super.close();
+    return new Callable<InputStream>() {
```

Comment:
I don't think that we need to wrap these calls anymore.

Reply:
woops, changing that and adding changelog entry